src/views/templates/details/tabs/scripts/tests/sshkey-utils.test.ts

```diff
@@ -0,0 +1,105 @@
+import { SecretModel, V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiCoreV1Secret } from '@kubevirt-ui/kubevirt-api/kubernetes';
+import { getTemplateVirtualMachineObject } from '@kubevirt-utils/resources/template';
+
+import { changeSSHKeySecret } from '../components/SSHKey/sshkey-utils';
+
+import { getMockTemplate } from './mocks';
+
+jest.mock('@openshift-console/dynamic-plugin-sdk', () => ({
+  k8sUpdate: jest.fn(({ data: template }) => Promise.resolve(template)),
+}));
+
+const template = getMockTemplate();
+const someExternalSecretName = 'some-external-secret';
+
+const sshkey = 'testtest';
```

Comment:
?

Reply:
Does not matter what string is. I just throw it some text